data/filedefs/filetypes.asciidoc

```diff
@@ -1,35 +1,33 @@
 # For complete documentation of this file, please see Geany's main documentation
 [styling]
-# no syntax highlighting yet
+# Edit these in the colorscheme .conf file instead
+default=default
+strong=default,bold
+emphasis=default,italic
+header1=keyword_1
+header2=keyword_1
+header3=keyword_1
+header4=keyword_1
+header5=keyword_1
+header6=keyword_1
+ulist_item=default
+olist_item=default
+blockquote=default,italic
+link=tag,bold
+code=default
+passthrough=default
+comment=comment
+literal=default
+attrib=default
+attribval=string
+macro=preprocessor
 
 [settings]
 # default extension used when saving files
-extension=asciidoc
+extension=adoc
```

Comment:
seems adoc is recommended by https://asciidoctor.org/docs/asciidoc-recommended-practices/, I don't know how "official" that source is.

Reply:
Thats as "official" as it gets for now, so fine.

Reply:
Accepted extensions for AsciiDoc files are: .adoc, .ad, .asc, .asciidoc

".adoc" is the actual recommended extension.